.travis.yml

```
@@ -22,6 +22,7 @@ jobs:
         - bash config/run-tests.sh
     - stage: lint
       script:
+        - export PYTHONPATH=./lab_"${lab}"
```

You cannot do it here since var `lab` is not defined.
Consider using the following at the top of `run-lint.sh` and `run-tests.sh` scripts:

```suggestion
export PYTHONPATH="$(pwd)/lab_1:$(pwd)/lab_2:$(pwd)/lab_3:$(pwd)/lab_4:$(pwd):${PYTHONPATH}"
```

---

done